src/components/contact-govuk-one-login/contact-govuk-one-login-controller.ts

```diff
@@ -1,7 +1,29 @@
 import { Request, Response } from "express";
+import { isValidUrl } from "./../../utils/strings";
+import pino from "pino";
 
-export function contactGet(req: Request, res: Response): void {
-  const data = {};
+const CONTACT_ONE_LOGIN_TEMPLATE = "contact-govuk-one-login/index.njk";
+const logger = pino();
 
-  res.render("contact-govuk-one-login/index.njk", data);
+export function contactGet(req: Request, res: Response): void {
+  const fromURL = getFromUrlAndSaveIt(req);
+  if (!fromURL) {
+    logger.info(
+      "Request to contact-govuk-one-login page did not contain a valid fromURL in the request or session"
+    );
+  }
+  const data = {
+    fromURL: fromURL,
```

Comment:
I think this can be simplified to 
`{ fromURL }`

Reply:
Resolved this in the next PR 
https://github.com/alphagov/di-account-management-frontend/pull/1058